src/App.tsx

```diff
@@ -53,7 +53,11 @@ function App() {
         {redirect && <Redirect to={redirect}></Redirect>}
         {/*  <React.StrictMode>*/}
         <FeatureToggleProvider
-          featureToggles={{[features.SURVEY_BUILDER]: true, [features.USERNAME_PASSWORD_LOGIN]: isDevelopment()}}>
+          featureToggles={{
+            [features.SURVEY_BUILDER]: true, 
+            [features.SURVEY_BRANCHING_LOGIC]: isDevelopment(),
+            [features.USERNAME_PASSWORD_LOGIN]: isDevelopment(),
```

Comment:
Do we still want to keep this hidden in production? Or does this not do what I think it does?

Reply:
The features flag allows you to turn features on/off. This just says that it should be "turned off" for production.

How it's actually used is that it is hidden as a "review your branching logic" on the completion page and that if you tab into the branching logic, it's read-only and there is a banner that tells you why its read-only.

Reply:
Actually, not sure what you are asking. I did not change `features.SURVEY_BUILDER` or `features.USERNAME_PASSWORD_LOGIN` - just reformatted for readability.

I added `features.SURVEY_BRANCHING_LOGIC` and set it to only read-write in staging.

Reply:
I guess I should have been more specific--I was asking if USERNAME_PASSWORD_LOGIN should still be marked as in development or if it's time to turn it on in production, especially now that the "forgot password" link is there.

Reply:
No, that is not production ready. We never resolved the issue of how to store the session token when the users hits refresh.

Reply:
:+1: